src/Common/Utilities/GeneralUtilities.cs

```diff
@@ -252,23 +252,27 @@ public static string EnsureTrailingSlash(string url)
             return address.Uri.AbsoluteUri;
         }
 
-        public static string GetHttpResponseLog(string statusCode, IDictionary<string, IEnumerable<string>> headers, string body)
+        public static string GetHttpResponseLog(string statusCode, IDictionary<string, IEnumerable<string>> headers, string body, IList<Regex> matchers = null)
```

Comment:
Keep original method and add new method with new input parameter

Reply:
this will be compatible with the original